### PR TITLE
Implement and Link ac_lotus

### PR DESCRIPTION
**Pull request by Hexalotl**
Adding in `ac_lotus`. Also realized I should be using the structure actor for lotus and koinobori.

### CODE REVIEW

File: src/ac_lotus.c [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  sizeof(LOTUS_ACTOR),
+  &aLOT_actor_ct,
+  &aLOT_actor_dt,
+  &aLOT_actor_init,
+  &aLOT_actor_draw,
+  NULL
+};
+
+StatusData_c lotus_StatusData = { 0, 0, 0, 0, 0x0A};
+ClObjPipeData_c lotus_CoInfoData = {
```
Comment (Prakxo) [inline src/ac_lotus.c:69]:
```
50] lotus_StatusData (object,local) found in ac_lotus.o 
50] lotus_CoInfoData (object,local) found in ac_lotus.o 
```
These are static

File: src/ac_lotus.c [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    {
+        70, // radius
+        6,  // height
+        0,  // offset
+
+        {0, 0, 0},  // center
+    },
+};
+
+ClObjPipe_c pipeinfo;
```
Comment (Prakxo) [inline src/ac_lotus.c:82]:
```suggestion
extern ClObjPipe_c pipeinfo;
```

File: src/ac_lotus.c [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+static void aLOT_setup_action(LOTUS_ACTOR* lotus, int action);
+static int aLOT_getPalNo();
+
+static void aLOT_actor_ct(ACTOR* actor, GAME* game) {
+    LOTUS_ACTOR* lotus;
+    int pal_no;
+    f32 water_height;
+
+    lotus = (LOTUS_ACTOR*)actor;
+    water_height = mCoBG_GetWaterHeight_File(actor->world.position, __FILE__, 0x109);
```
Comment (Prakxo) [inline src/ac_lotus.c:94]:
```suggestion
    water_height = mCoBG_GetWaterHeight_File(actor->world.position, __FILE__, 265);
```

File: src/ac_lotus.c (hunk 2 above)
Comment (Cuyler36) [reply src/ac_lotus.c]: This is static, not external.
image

Comment (Cuyler36) [reply src/ac_lotus.c]: Actually, it is external. It's only accessed locally though. Sorry.

**Pull request merged**